beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -455,7 +461,14 @@ func (s *Server) GetAttesterSlashings(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	sourceSlashings := s.SlashingsPool.PendingAttesterSlashings(ctx, headState, true /* return unlimited slashings */)
-	slashings := structs.AttesterSlashingsFromConsensus(sourceSlashings)
+	ss := make([]*eth.AttesterSlashing, 0, len(sourceSlashings))
+	for _, slashing := range sourceSlashings {
+		s, ok := slashing.(*eth.AttesterSlashing)
+		if ok {
```

Comment:
should we add a comment or something handling the not ok situations?

Reply:
I think we need to handle every case where it's not OK

Reply:
I fixed this.